src/pageLayout/constants/navigationHierarchy.ts

```diff
@@ -216,5 +216,17 @@ export const generateNavItems = (orgID: string): NavItem[] => {
         },
       ],
     },
+    {
```

Comment:
@hoorayimhelping found another giant config object! not advocating for using context on this, but maybe it'd save some time if we added routes to this object and defaulted to lazy loading them?

Reply:
the SetOrg looks repetitive